### PR TITLE
[TOS-965]fix: fixing 'variable' spelling in store nlp

### PR DESCRIPTION
**Pull request by nixonakash01**
Jira Ticket: https://testsigma.atlassian.net/browse/TOS-965
Github Issue: https://github.com/testsigmahq/testsigma/issues/244
What is the current behavior?
getting this step: Store current Test... value into runtime varible attribute

What is the expected behavior?
should be: Store current Test... value into runtime variable attribute

For the NLP
'Store current  ${Test Data Profile Name/Test Data Set Index/Test Data Set Name} value into runtime varible  @{attribute' 
The spelling for variable was changed from 'varible' to 'variable'.

### CODE REVIEW

File: server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE `natural_text_actions` SET `natural_text` = 'Store current  ${Test Data Profile Name/Test Data Set Index/Test Data Set Name} value into runtime variable  @{attribute}' WHERE id = 1095;
```
[the remark this answers is not in the document]
Comment (nixonakash01) [reply server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql:1]: Okay sure, will look into it

[the remark this answers is not in the document]
Comment (nixonakash01) [reply server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql:1]: @tarun-testsigma Have made changes in the bootstrap. should i add migration for the 4 changes and should i use update or insert for that command?

[the remark this answers is not in the document]
Comment (nixonakash01) [reply server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql]: @tarun-testsigma bro, i have added the migration for both the tables. can you verify them in the latest commit?

File: server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+UPDATE `natural_text_actions` SET `natural_text` = 'Store current  ${Test Data Profile Name/Test Data Set Index/Test Data Set Name} value into runtime variable  @{attribute}' WHERE id = 10209;
+UPDATE `natural_text_actions` SET `natural_text` = 'Store current  ${Test Data Profile Name/Test Data Set Index/Test Data Set Name} value into runtime variable  @{attribute}' WHERE id = 20163;
+UPDATE `natural_text_actions` SET `natural_text` = 'Store current  ${Test Data Profile Name/Test Data Set Index/Test Data Set Name} value into runtime variable  @{attribute}' WHERE id = 30192;
+UPDATE `natural_text_action_examples` SET `description` = 'Store current Test Data Profile Name/Test Data Set Index/Test Data Set Name value into runtime variable' WHERE id = 854;
```
Comment (tarun-testsigma) [inline server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql:5]: @nixonakash01 Have you tested these changes ?

File: server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql (hunk 1 above)
[the remark this answers is not in the document]
Comment (tarun-testsigma) [reply server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql]: @nixonakash01 Looks good. Have you tested these changes ?

[the remark this answers is not in the document]
Comment (nixonakash01) [reply server/src/main/resources/db/migration/V52__edit_spelling_in_nlp.sql]: yes. i changed the corresponding values in database and verified the spelling in web, mobile web, ios and android @tarun-testsigma

**Pull request merged**